vars/capsuleSystemTests.groovy

```diff
@@ -85,8 +87,11 @@ void call(Map additionalConfig) {
   }
   
   stage('start nomad') {
+    dir('system-tests'){
+        sh 'cp ./vegacapsule/nomad_config.hcl' + ' ' + testDirectoryPath + '/nomad_config.hcl'
+    }
     dir ('tests') {
-        sh 'daemonize -o ' + testDirectoryPath + '/nomad.log -c ' + testDirectoryPath + ' -p ' + testDirectoryPath + '/vegacapsule_nomad.pid ' + testDirectoryPath + '/vegacapsule nomad'
+        sh 'daemonize -o ' + testDirectoryPath + '/nomad.log -c ' + testDirectoryPath + ' -p ' + testDirectoryPath + '/vegacapsule_nomad.pid ' + testDirectoryPath + '/vegacapsule nomad --nomad-config-path=' + testDirectoryPath + '/nomad_config.hcl'
```

Comment:
Just one question why we are going to use a specific nomad config? :) 

I mean this file: Just one question why we are going to use a specific nomad config? :)

Reply:
oh nevermind, I see why, because default config limits resources